src/ripple/app/paths/AMMOffer.h

```diff
@@ -103,15 +104,19 @@ class AMMOffer
     limitOut(
         TAmounts<TIn, TOut> const& offrAmt,
         TOut const& limit,
-        bool fixReducedOffers,
+        Rules const& rules,
```

Comment:
Consider using the globally available rules rather than passing this as a parameter. (also fine as-is, but I think I'd like to get away from passing rules around as much as we do and just using the new global).